src/main/kotlin/leets/leenk/domain/notification/infrastructure/NotificationPublisher.kt

```
@@ -0,0 +1,46 @@
+                fcmToken,
+                notification.notificationType.path,
+                user.id,
+            )
+        eventPublisher.publishEvent(sqsEvent)
+    }
+
+    private fun User.getFcmTokenReflection(): String? =
+        try {
+            val method = this.javaClass.getMethod("getFcmToken")
+            method.invoke(this) as? String
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)